<ui version="4.0" >
 <class>ManagePluginsGui</class>
 <widget class="QDialog" name="ManagePluginsGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Visual Netkit - Manage Plugins</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Manage Plugins</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Select only plugins that you want connect</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="pluginsList" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title" >
         <string>Selected plugin info</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="leftMargin" >
          <number>3</number>
         </property>
         <property name="topMargin" >
          <number>3</number>
         </property>
         <property name="rightMargin" >
          <number>3</number>
         </property>
         <property name="bottomMargin" >
          <number>3</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pName" >
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string>Version:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pVersion" >
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string>Desription:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pDescription" >
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string>Dependencies:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pDeps" >
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string>Authors:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pAuthor" >
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManagePluginsGui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManagePluginsGui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
